clients/vault/src/system.rs

```diff
@@ -150,11 +156,32 @@ impl VaultIdManager {
 		self.vault_data.read().await.get(vault_id).map(|x| x.stellar_wallet.clone())
 	}
 
+	pub async fn get_active_vault(&self, vault_id: &VaultId) -> Option<VaultData> {
```

Comment:
The reason I added this method is to have it into account for the future. Now, `get_vault` will bring the data even if the vault is liquidated. This impacts a few processes, namely:
- [Redeem](https://github.com/pendulum-chain/spacewalk/blob/7c7989875e95e1cfe3b0aeba9bb6af01d9e33e58/clients/vault/src/redeem.rs#L26).
- [Pay and Execute](https://github.com/pendulum-chain/spacewalk/blob/7c7989875e95e1cfe3b0aeba9bb6af01d9e33e58/clients/vault/src/requests/execution.rs#L242)
- [Accept and replace](https://github.com/pendulum-chain/spacewalk/blob/7c7989875e95e1cfe3b0aeba9bb6af01d9e33e58/clients/vault/src/replace.rs#L38)

I think these are all processes that the liquidated vault should be able to perform anyway. This is the most critical difference about this change, simply replacing in those instances with `get_active_vault` will not modify the functionality, in case this assumptions are wrong.